ValveResourceFormat/ThirdParty/DDSImage.cs

```diff
@@ -229,47 +229,76 @@ private static void DecompressBlockDXT5(int x, int y, int width, byte[] blockSto
                         }
                     }
 
-                    byte colorCode = (byte)((code >> 2 * (4 * j + i)) & 0x03);
-
+                    byte positionCode = (byte)((code >> 2 * (4 * j + i)) & 0x03);
                     int finalR = 0, finalG = 0, finalB = 0;
 
-                    switch (colorCode)
+                    if (color0 > color1)
```

Comment:
Rather than having an if statement with two switch cases, couldn't the if statement just increment positionCode by 4, and have:
```csharp
switch(positionCode)
{
    case 0:
    case 4:
        //INSERT LOGIC
        break;
    case 1:
    case 5:
        //insert logic
        break
    case 2:
        break
    case 3:
        break
    case 6:
        break
    case 7:
        break
}
```